### ISSUE TITLE
Should prevent hidden files like .DS_Store

### ISSUE THREAD

**Issue by hildevestol** [hyperoslo/heroku-pages#7]
On OSX we usually get a `.DS_Store` file in folders.. this probably shouldn't get sent to heroku. Perhaps ignore all files that start with `.`?

This is what I got on upload:

````
heroku pages:upload -r heroku
upload: public/heroku_pages/500.html to s3://my_domain/heroku_pages/500.html
upload: public/heroku_pages/.DS_Store to s3://my_domain/heroku_pages/.DS_Store
upload: public/heroku_pages/images/rema-logo.png to s3://my_domain/heroku_pages/images/rema-logo.png
````

Otherwise.. Love the plugin :+1:  :smile:

**Comment by sindrenm** [comment]
Good one! :+1:

**Comment by hildevestol** [comment]
Fixed by PR #8
